### PR TITLE
Add no-multiple-var rule

### PR DESCRIPTION
**Pull request by YuichiNukiyama**
related issue #525.

- for another linter compatibility, I define target options (`var`,
`let` and `const`)
- I check `for statement`, but don't check `for...in statement` and
`for...of statement`. becase, TS disallow multiple variable definitions
in the same statement by default

### CODE REVIEW

**Comment by JKillian** [comment]
Thanks for the PR @YuichiNukiyama!

> for another linter compatibility, I define target options (var, let and const)
I'm not sure I understand why we need these options. I think this rule could maybe be simplified and not have any options? What do you think?

**Comment by YuichiNukiyama** [comment]
@JKillian If you prefer simplicity to compatibility, I remove options (use only `true` option) and validate all of them (`var`, `let` and `const`) together.

**Comment by JKillian** [comment]
@YuichiNukiyama No options seems best to me in this case. In what scenarios do you think people might want the different options?

**Comment by YuichiNukiyama** [comment]
Speaking honestly, I don't want these options. So, I follow your idea.

**Comment by JKillian** [comment]
Looks a lot better, thanks @YuichiNukiyama! One more idea, we might want an "ignore-for-loop" option to make the rule not check for loop declarations?

File: README.md [hunk 1]
```diff
@@ -242,6 +242,8 @@ A sample configuration file with all options is available [here](https://github.
 * `no-internal-module` disallows internal `module` (use `namespace` instead).
 * `no-invalid-this` disallows using the `this` keyword outside of classes.
     * `no-this-in-function-in-method` disallows using the `this` keyword in functions within class methods.
+* `no-multiple-var` disallows multiple variable definitions in the same statement.
+* `no-use-before-declare` disallows usage of variables before their declaration.
```
Comment (JKillian) [inline README.md]: Also, can you delete this line?

**Comment by YuichiNukiyama** [comment]
@JKillian `ignore-for-loop` option is good idea.  I' ll send new issue and PR.

**Pull request merged**